lib/minasu/db.py

```diff
@@ -12,13 +14,22 @@ def __init__(
         self.instance_dir  = instance_dir
         self.instance_path = self.instance_dir + "/" + self.instance_name
 
+        program = (instance_name in (p.name() for p in psutil.process_iter()))
+        lock_dir = '/tmp/minasu/' + instance_name
+        if os.path.exists(lock_dir) or program:
+            print("Minasu instance: %s, is already running" % instance_name)
+            print("Exiting")
+            sys.exit()
+        else:
+            os.mkdir(lock_dir)
```

Comment:
the lock should only happen in the load method, not in the __init__ method.

Reply:
I think we should know if there is a existing DB running as soon as the new one is instantiated.

Reply:
Nope.  See the design doc.  The instantiation only gives you a usable object.  You must then call load with the name and path of an instance.  You can't lock a dir in a path you don't even have yet.